sdk_container/src/third_party/coreos-overlay/sec-policy/selinux-base/selinux-base-2.20221101-r4.ebuild

```diff
@@ -29,34 +28,27 @@ HOMEPAGE="https://wiki.gentoo.org/wiki/Project:SELinux"
 LICENSE="GPL-2"
 SLOT="0"
 
-# flatcar changes
 RDEPEND=">=sys-apps/policycoreutils-2.8
-	>=sys-apps/checkpolicy-2.8
-"
+	>=sys-apps/checkpolicy-2.8"
 DEPEND="${RDEPEND}"
```

Comment:
The sys-apps/checkpolicy is originally placed in BDEPEND, why moving it back to RDEPEND?